twindle-web/team_details/README.md

```diff
@@ -4,32 +4,32 @@ This page contains the list of Twindle Teammates that our working & contributing
 
 ## How to add your name to the list
 
-To add your name to the list, simply open `persons.js` file, and then add your own row at the bottom, using the below template:
+To add your name to the list, simply open `data.json` file, and then add your own data at the bottom, using the below template:
 ( **Don't include the square brackets "[ ]" !!! They are just there to indicate an example placeholder.** )
 
 ```moonscript
```

Comment:
what's this

Reply:
@johnjacobkenny I have updated the Readme file because now we are using a `data.json` file to retrieve the users data for the card instead of `persons.js` file. 

We are using a json object in which every person has to fill their data for the generation of the card according to the template mentioned here in the readme file. This is the fastest way to retrieve & display data rather than using github API

Reply:
Yes, my doubt is about the **moonscript**